app/controllers/CreateGroupSelectTeamMembersController.scala

```diff
@@ -84,19 +86,21 @@ class CreateGroupSelectTeamMembersController @Inject()
           .bindFromRequest()
           .fold(
             formWithErrors => {
-              teamMemberService.getPageOfTeamMembers(arn)(1, 10).flatMap(paginatedList =>
-                sessionCacheService.get(RETURN_URL).map(returnUrl =>
-                  Ok(
-                    select_paginated_team_members(
-                      paginatedList.pageContent,
-                      groupName,
-                      backUrl = Some(returnUrl.getOrElse(routes.CreateGroupSelectClientsController.showReviewSelectedClients(None, None).url)),
-                      form = formWithErrors,
-                      paginationMetaData = Some(paginatedList.paginationMetaData)
+              teamMemberService
+                .getPageOfTeamMembers(arn)(1, 10)
+                .flatMap(paginatedList =>
+                  sessionCacheService.get(RETURN_URL).map(returnUrl =>
```

Comment:
FYI we can get rid of RETURN_URL for paginated since we don't have a CYA screen with change links anymore